frontend/src/metabase/containers/SaveQuestionModal/SaveQuestionModal.tsx

```diff
@@ -113,10 +113,21 @@ export const SaveQuestionModal = ({
     }
   }
 
+  const getOriginalNameModification = (originalQuestion: Question | null) =>
+    originalQuestion
+      ? originalQuestion?.displayName() + " - " + t`Modified`
```

Comment:
This won't work nicely for RTL languages.
We need to make the question name a part of the translation string, something like
```tsx
t`${questionName} - Modified`
```